shared/whats-new/index.tsx

```diff
@@ -0,0 +1,156 @@
+          <Kb.Box2
+            direction="vertical"
+            alignItems="flex-start"
+            alignSelf="flex-start"
+            style={styles.scrollViewInner}
+          >
+            {children}
+          </Kb.Box2>
+        </Kb.ScrollView>
+      </Kb.Box2>
+    </Kb.Box2>
+  ) : (
+    <Kb.Box2
+      direction="vertical"
+      alignItems="flex-start"
+      alignSelf="flex-start"
+      fullHeight={true}
+      style={styles.popupContainer}
+    >
+      <Kb.Box2
+        direction="vertical"
+        alignItems="flex-start"
+        alignSelf="flex-start"
+        fullHeight={true}
+        fullWidth={true}
+        style={styles.contentBackground}
+      >
+        <Kb.ScrollView style={styles.scrollView}>{children}</Kb.ScrollView>
+      </Kb.Box2>
+    </Kb.Box2>
+  )
+
+class WhatsNew extends React.PureComponent<Props> {
+  static navigationOptions = {}
+  componentWillUnmount() {
+    this.props.onBack()
+  }
+
+  render() {
+    const {
+      currentVersion,
+      lastVersion,
+      lastLastVersion,
+      noVersion,
+      Current,
+      Last,
+      LastLast,
+      seenVersions,
+      onNavigate,
+      onNavigateExternal,
+    } = this.props
+    return (
+      <Wrapper>
+        {Current && (
+          <Current
+            seen={seenVersions[currentVersion]}
+            onNavigate={onNavigate}
+            onNavigateExternal={onNavigateExternal}
+          />
+        )}
+        {lastVersion && lastVersion !== noVersion && Last && (
+          <Last
+            seen={seenVersions[lastVersion]}
+            onNavigate={onNavigate}
+            onNavigateExternal={onNavigateExternal}
+          />
+        )}
+        {lastLastVersion && lastLastVersion !== noVersion && LastLast && (
+          <LastLast
+            seen={seenVersions[lastLastVersion]}
+            onNavigate={onNavigate}
+            onNavigateExternal={onNavigateExternal}
+          />
+        )}
+      </Wrapper>
+    )
+  }
+}
+
+const modalWidth = 288
+const modalHeight = 424
+const styles = Styles.styleSheetCreate(() => ({
+  contentBackground: Styles.platformStyles({
+    common: {
+      backgroundColor: Styles.globalColors.blueGrey,
+      ...Styles.globalStyles.rounded,
+    },
+    isElectron: {
+      // Align menu edge with icon on desktop
+      marginRight: Styles.globalMargins.xtiny,
+    },
+  }),
+  popupContainer: Styles.platformStyles({
+    isElectron: {
+      height: modalHeight,
+      maxHeight: modalHeight,
+      maxWidth: modalWidth,
+      width: modalWidth,
+    },
+  }),
+  scrollView: Styles.platformStyles({
+    common: {
+      width: '100%',
+    },
+    isElectron: {
+      ...Styles.padding(Styles.globalMargins.tiny),
+    },
+  }),
+  scrollViewInner: Styles.platformStyles({
+    isMobile: {
+      marginBottom: Styles.globalMargins.small,
+      marginLeft: Styles.globalMargins.small,
+      marginRight: Styles.globalMargins.small,
+      marginTop: Styles.globalMargins.small + 2,
```

Comment:
We should avoid doing that. Did you especially need 18px? or could 16 work?

Reply:
16 is correct. Was actually trying to match the mockup which has 18 to the top. Forgot to follow up about that since you were out last week, thanks for catching this.

Reply:
Oh likely when you see 18 on a mockup it's a mistake and it should be rounded to 16.